wiki/Skinning/History/en.md

```diff
@@ -13,9 +14,9 @@ The skinning elements listed here are no longer in use.
 
 ![](img/menu-osu.png)
 
-| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
-|:-:|:-:|:-:|:-:|:-:|:-:|
-| All | ![No][false] | ![Yes][true] | Normal | Centre | - |
+| Versions |  Animatable  | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
```

Comment:
The block tag was added because these tables must not be beautified or have these extra spaces and marks to make them line up. See the tables section in the ASC for more details. I can also apply the changes if you don't get to it first.

Reply:
Please go ahead and fix it, guess I had prettier enabled by accident